src/Symfony/Component/Mailer/Command/DebugCommand.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Mailer\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Mailer\Transport\Transports;
+
+/**
+ * A console command to display Symfony Mailer transports on the command line.
+ *
+ * @author Fritz Michael Gschwantner <fmg@inspiredminds.at>
+ */
+final class DebugCommand extends Command
+{
+    protected static $defaultName = 'mailer:debug';
```

Comment:
```suggestion
    protected static $defaultName = 'debug:mailer';
```

To be consistent with all other debug commands that are under the `debug` namespace.

Reply:
I still don't think this is really needed as all the information is already in `config/packages/mailer.yaml`.
As a reminder, we are trying to limit the number of commands as the list is already quite long. So, we should be careful to only add high-value commands. I don't think this one qualifies, at least not as is.

Reply:
> I still don't think this is really needed as all the information is already in config/packages/mailer.yaml.

Any bundle can add additional configs not visible to the user otherwise. But other than that I agree, `debug:mailer` is not really a must have.

Reply:
Let's remove it for now and let's be focus on the other command, that will help merge this PR faster.